rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,211 @@
+- Research and prototyping tasks SHOULD be modeled as Spikes.
+- Dependencies between components within a Sprint MUST be communicated between the respective teams and surfaced using [ZenHub Dependencies](https://help.zenhub.com/support/solutions/articles/43000010349). 
+
+At the start of the Sprint, the single DCP Sprint Backlog SHOULD be reviewed by the **PM Team** and **Architecture Team** in their oversight roles.
+
+### Sprint(ing)
+
+During Development, individual developers assign items from the ZenHub *Sprint Backlog* pipeline and move to the *In Progress* pipeline. 
+
+When there is a pull request in review for an *In Progress* item, the item is moved to the *QA/Review* pipeline. 
+
+If rejected, the item returns to *In Progress*; otherwise, it is deployed in Dev for verification and moves to the *Done* pipeline.
+
+Items that are *Done* are moved to the *Closed* pipeline when verified in Dev or Integration. 
+
+During Sprints, the **Architecture Team** MAY schedule Scrums on a regular cadence for the DCP Development Team *"to identify if any integration issues exist."* (See *Daily Scrum* in [*Scrum Nexus*](http://scrumorg-website-prod.s3.amazonaws.com/drupal/2018-01/2018-Nexus-Guide-English_0.pdf) for guidance)
+
+THe **PM Team** monitors progress using ZenHub charts and reports.
+
+### Sprint Review
+
+**Every four weeks**. On the penultimate day of the Sprint, there is a one hour meeting for the Sprint Review open to the entire DCP commmunity led by the **PM Team**. Major product increments selected by **Technical Leads** are demonstrated to key stakeholders.
+
+(See *Sprint Review* in [*Scrum Guide*](https://scrumguides.org/docs/scrumguide/v2017/2017-Scrum-Guide-US.pdf) and *Nexus Sprint Review* in [*Nexus Guide*](http://scrumorg-website-prod.s3.amazonaws.com/drupal/2018-01/2018-Nexus-Guide-English_0.pdf) for guidance.)
+ 
+### Sprint Retrospective
+
+**Every four weeks**. On the final day of the Sprint, there is a one hour meeting for the Sprint Retrospective open to the entire DCP commmunity led by the **PM Team**.
+ 
+The Scrum Retrospective identifies opportunities to improve the planning process in future iterations. 
+
+(See *Sprint Retrospective* in [*Scrum Guide*](https://scrumguides.org/docs/scrumguide/v2017/2017-Scrum-Guide-US.pdf) and *Nexus Sprint Retrospective* in [*Nexus Guide*](http://scrumorg-website-prod.s3.amazonaws.com/drupal/2018-01/2018-Nexus-Guide-English_0.pdf) for guidance.)
+
+### Acceptance Criteria
+
+This process will be under continuous community introspection and improvement through the Sprint Retrospective. If the process fails to deliver product increments with high confidence after three Sprints, then more radical alternatives should be pursued that satisfy the requirements from our key stakeholders in HCA Science Governance. We should also assess whether ZenHub is meeting our needs for a common project management application. 
```

Comment:
What metrics do you propose to measure delivery?

Reply:
1. There is small set of DCP-level priorities that we're all aligned to. There is less distraction and time-slicing required due to "all features that are important to someone are equal and must now be simultaneously designed".  For example, if DCP Pri 0 is _Awesome Metadata Consumer_ solution, then our attention is on _Awesome Metadata Consumer_ solution and not on Favorite-Feature-A and Feature-Feature-B. In the absence of quarterly roadmaps, this will be a challenging conversation at first.
2. Teams are having more open conversations about their struggles balancing DCP-level priorities with their DCP-component priorities.
3. Estimation is improving. We have high(er) confidence in our ability to deliver on a quarterly basis as desired by _Oversight_.  This requires teams to be more introspective to be successful. 
4. The planning process is responsive to spontaneous and Retrospective feedback. 

We could also gauge community satisfaction with a survey.